omc3/check_corrections.py

```diff
@@ -462,7 +463,7 @@ def _get_measurement_filter(nominal_model: TfsDataFrame, opt: DotDict) -> dict[s
 
 def _create_model_and_write_diff_to_measurements(
         output_dir: Path, measurement: OpticsMeasurement, correction_name: str, correction_files: Sequence[Path],
-        accel_inst: Accelerator, rms_masks: dict) -> OpticsMeasurement:
+        accel_inst: Accelerator, rms_masks: dict, variable_categories: Sequence[str]) -> OpticsMeasurement:
```

Comment:
Does that make sense to add here? As far as I can see this is only called with an empty array. What is your plan for this? Adding this as optional input argument to check how well  the correction of DPP will work?